tsconfig.json

```diff
@@ -28,6 +28,7 @@
   "include": [
     "./src/**/*.js",
     "./src/**/*.ts",
-    "./src/**/*.d.ts"
+    "./src/**/*.d.ts",
+    "./typings.d.ts"
```

Comment:
It seems that this doesn't affect the build at all.
There is some overlapping with `common-types.d.ts` and some differences for the same stuff as well.